### PR TITLE
UAR-1590 / UAR-1631 Add new BO individual NOCs for update/remove journeys

### PR DESCRIPTION
**Pull request by markpit**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-1590
https://companieshouse.atlassian.net/browse/UAR-1631 (subtask)

### Change description
Amend the Natures of Control for ‘Tell Us About the Individual’ and ‘Review the Individual Beneficial Owner’ Pages
The new NOCs being added are controlled by the feature flag FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/includes/page-templates/beneficial-owner-nature-of-control-types.html [hunk 1]
```diff
@@ -14,7 +14,8 @@
   {% include "includes/inputs/nature-of-control-checkboxes/trustees-nature-of-control-types.html" %}
 {% endif %}
 
-{% if FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC and not entity_number %}
+{# if feature flag on, show following for all registration BOs and BO Individual on the update/remove journey#}
+{% if FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC and ((not entity_number) or isBeneficialOwnerIndividual) %}
```
Comment (lduranteau) [inline views/includes/page-templates/beneficial-owner-nature-of-control-types.html:18]: Why did you add this condition?
and ((not entity_number) or isBeneficialOwnerIndividual)

Just remove:
and not entity_number

Comment (lduranteau) [reply views/includes/page-templates/beneficial-owner-nature-of-control-types.html:18]: Mine are ready and we can merge them at the same time.

In any case, as long as you don't include ‘beneficial-owner-nature-of-control-jurisdictions.html’ in the view, they won't be displayed.

Comment (markpit) [reply views/includes/page-templates/beneficial-owner-nature-of-control-types.html:18]: ok, it makes sense to remove it, we are making the changes at the same time anyway. I'll update the html.

Comment (markpit) [reply views/includes/page-templates/beneficial-owner-nature-of-control-types.html]: updated

**Review by lduranteau — APPROVED** [review]

File: views/update/review-beneficial-owner-individual.html [hunk 2]
```diff
@@ -61,7 +61,9 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Review the individual beneficial o
       {% include "includes/inputs/fields/is-still-a-beneficial-owner-input.html" %}
 
       {% set isBeneficialOwnerGov = false %}
+      {% set isBeneficialOwnerIndividual = true %}
```
Comment (lduranteau) [inline views/update/review-beneficial-owner-individual.html:64]: You forgot to remove this variable

**Review by mwestacott — COMMENTED** [review]
LGTM. Just that minor spot from @lduranteau.

File: views/update/review-beneficial-owner-individual.html (hunk 2 above)
Comment (markpit) [reply views/update/review-beneficial-owner-individual.html]: updated

**Review by mwestacott — APPROVED** [review]
